src/javascripts/components/copy.js

```diff
@@ -0,0 +1,21 @@
+;(function (global) {
+  'use strict'
+  var $ = global.jQuery
+  var GOVUK = global.GOVUK || {}
+
+  // This module is dependent on /vendor/clipboard.js
+  GOVUK.copy = {
+    init: function (selector) {
+      $(selector).parent().prepend('<a class="govuk-c-link app-c-link--copy" href="#copy">Copy</a>')
+      // Copy to clipboard
+      new Clipboard('.app-c-link--copy', {
+        target: function (trigger) {
+          return trigger.nextElementSibling
+        }
```

Comment:
Maybe we could use try/catch here for error checking, or check if `Clipboard` exists? This throws up a fatal error if for any reason `vendor/clipboard.js` doesn't load.

Reply:
Added